src/mlpack/core/optimizers/cne/cne_impl.hpp

```diff
@@ -0,0 +1,220 @@
+/**
+ * @file cne_impl.hpp
+ * @author Marcus Edel
+ * @author Kartik Nighania
+ *
+ * Conventional Neural Evolution
```

Comment:
Can you elaborate on this, a simple sentence what this method does is just fine.